scripts/gitlab.py

```diff
@@ -0,0 +1,26 @@
+from requests import request
+from json import dumps
+
+
+def query_gitlab(url, projectid, token):
```

Comment:
Consider writing a single function as a clean interface to get the descriptor from a project.

get_gantt_descriptor_from_gitlab_proj(gitlab_url, project_id, token)
and simply return the json